### PR TITLE
UIP-961: move avatar storybook documentation over to csf

### PR DESCRIPTION
**Pull request by reywright**
## Description
Move Avatar over to csf style documentation.
I'd like a review mostly just to validate that this makes sense as I'll be doing it for more components.

## Screenshots
Screen Shot 2020-10-13 at 9 30 55 PM

## Checklist
N/A

### CODE REVIEW

File: .storybook/preview-head.html [hunk 1]
```diff
@@ -354,7 +354,7 @@
     color: #333;
     font-size: 14px;
     cursor: pointer;
-    font-family: "Operator Mono","Fira Code Retina","Fira Code","FiraCode-Retina","Andale Mono","Lucida Console",Consolas,Monaco,monospace;
+    font-family: "Nunito Sans",-apple-system,".SFNSText-Regular","San Francisco",BlinkMacSystemFont,"Segoe UI","Helvetica Neue",Helvetica,Arial,sans-serif;
```
Comment (reywright) [inline .storybook/preview-head.html:357]: Changing the font style of the import statement more closely match the title.

File: .storybook/preview.js [hunk 2]
```diff
@@ -1,49 +1,56 @@
...
-    { name: "White", value: FDS.COLOR_WHITE },
-    { name: "Haze", value: FDS.COLOR_HAZE},
-    { name: "Navy", value: FDS.COLOR_NAVY }
-  ],
+export const parameters = {
+  docs: {
+    page: () => (
+      <>
+        <Title />
+        <ImportPath2 />
```
Comment (reywright) [inline .storybook/preview.js:22]: This is the only additional thing we're adding to the doc page, an import statement right below the title.

File: src/util/storybook-docs/ImportPath2.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import { DocsContext } from '@storybook/addon-docs/blocks';
+import CloneIcon from 'lib/icons/react/CloneIcon';
+import CheckIcon from 'lib/icons/react/CheckIcon';
+import useToggleHover from './useToggleHover';
+import useClipboard from './useClipboard';
+
+export const ImportPath = () => {
+  const toggleHover = useToggleHover()[1];
+  const [copiedText, copyToClipboard] = useClipboard();
+  const context = useContext(DocsContext);
```
Comment (reywright) [inline src/util/storybook-docs/ImportPath2.jsx:11]: This is all that's needed to pull in storybook context.

File: src/util/storybook-docs/ImportPath2.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+  const componentName = context.kind.split('/')[1];
```
Comment (reywright) [inline src/util/storybook-docs/ImportPath2.jsx:13]: I'm merely taking `components/Avatar` and just taking `Avatar`. We know the rest of the path.

**Review by akdetrick — APPROVED** [review]
changes LGTM - how do you want to roll this out to other components?

**Comment by reywright** [comment]
@akdetrick 

I figure one big PR (this one). It'll take me around 2 days to complete it. I would do smaller PR's but

1. I figure we're not modifying production code
2. It should be easy to walk through production and local documentation side by side to verify
3. I can roll out slowly but if you check this out locally, I broke knobs by doing this.
4. I could maybe get knobs and controls to co-exist but it creates a muddy narrative: "hey everybody we migrated only certain stories so only these stories will be completely different" vs "we are now fully on controls, this is how you use them now".

So yeah current plan is one big PR and then we can spend some quality time reviewing (and also load up on sweet.imfast.io so that our designers can review as well)

**Review by akdetrick — APPROVED** [review]
Love the centrally defined page template. I think the changes involved make sense

**Comment by reywright** [comment]
Closing in favor of CSF release branc

**Pull request closed without merging**